tasks/managed/push-rpm-to-koji/push-rpm-to-koji.yaml

```diff
@@ -29,35 +29,48 @@ spec:
     - name: pipelineImage
       type: string
       description: |
-        The image url with koji, kinit installed for running the push-rpm-to-koji task,
-        please make sure you have such image or you build this image first.
+        The image url with koji (1.34 or higher), jq and kinit installed for
+        running the push-rpm-to-koji task, please make sure you have such image
+        or you build this image first.
   steps:
     - name: push-rpm-to-koji
       # The pipelineImage will be fetching from get-secrets task with koji installed there
       image: "$(params.pipelineImage)"
+      env:
+        - name: SNAPSHOT_SPEC_FILE
+          value: $(params.snapshotPath)
+        - name: DATA_FILE
+          value: $(params.dataPath)
       script: |
         #!/usr/bin/env bash
         set -eux
 
-        SNAPSHOT_SPEC_FILE="$(params.snapshotPath)"
         if [ ! -f "${SNAPSHOT_SPEC_FILE}" ] ; then
```

Comment:
This isn't going to work. The pipeline is receiving the collect-data result as a parameter. That is a relative path inside the workspace. You don't add the workspace here, so it won't work

Reply:
Isn't the current directory the workspace? That's why I needed to use the `readlink` below. Also, in tests the path is absolute, so settings `SNAPSHOT_SPEC_FILE=$(workspaces.data.path)/$(params.snapshotPath)` won't work.

Reply:
Ack. I've changed the task params back to relative paths. But I still use `env` to pass the values to the script - using `$(...)` can be unsafe (script injection).